etna/core/models/mixins.py

```diff
@@ -199,6 +201,36 @@ class Meta:
     base_form_class = RequiredHeroImagePageForm
 
 
+class AccentColourMixin(models.Model):
+    """Mixin to add accent_colour attribute to a Page."""
+
+    accent_colour = models.CharField(
```

Comment:
This should instead be a dropdown of:

- none
- black
- pink
- orange
- yellow
- green
- blue